lrelease.pri

```diff
@@ -0,0 +1,36 @@
+# Copyright 2005-2017 The Mumble Developers. All rights reserved.
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file at the root of the
+# Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+# This .pri files finds a suitable protoc binary on the system.
```

Comment:
The whole comment block is wrong. It talks about protoc.

Just do a simple description of what this pri files does, instead.
No need for a big one like for the other scripts that are more complicated.